pkgdb2/templates/list_actions.html

```diff
@@ -58,7 +58,7 @@
     <tr>
         <td>
         {% if page > 1%}
-            <a href="{{ url_for('.admin_action') }}?page={{page - 1}}&package={{package}}&from_date={{from_date}}">
+            <a href="{{ url_for('.admin_actions') }}?page={{page - 1}}&package={{package}}&from_date={{from_date}}">
```

Comment:
No, I won't.  At least not yet.  ;)  I'm going to write test coverage for this first to be sure that further changes won't break it again.